boards/lilygo/ttgo_t8c3/ttgo_t8c3.dts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2024 Lothar Felten <lothar.felten@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <espressif/esp32c3/esp32c3_common.dtsi>
+#include "ttgo_t8c3-pinctrl.dtsi"
+
+/ {
+	model = "Lilygo ttgo t8-c3";
+	compatible = "lilygo,ttgo-t8c3";
+
+	aliases {
+		led0 = &green_led;
+		i2c-0 = &i2c0;
+		watchdog0 = &wdt0;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,console = &usb_serial;
+		zephyr,shell-uart = &usb_serial;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,canbus = &twai;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led: led_0 {
+			gpios =  <&gpio0 3 GPIO_ACTIVE_HIGH>;
+			label = "Green - LED0";
+		};
+	};
+};
+
+&uart0 {
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart0_default>;
+	pinctrl-names = "default";
+};
+
+&usb_serial {
+	status = "okay";
+};
+
+&i2c0 {
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_STANDARD>;
+	pinctrl-0 = <&i2c0_default>;
+	pinctrl-names = "default";
+};
+
+&trng0 {
+	status = "okay";
+};
+
+&spi2 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+	status = "okay";
+	pinctrl-0 = <&spim2_default>;
+	pinctrl-names = "default";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&wdt0 {
+	status = "okay";
+};
+
+&timer0 {
+	status = "okay";
+};
+
+&timer1 {
+	status = "okay";
+};
+
+&twai {
+	status = "okay";
+	pinctrl-0 = <&twai_default>;
+	pinctrl-names = "default";
+};
+
+&wifi {
```

Comment:
This will/might conflict with https://github.com/zephyrproject-rtos/zephyr/pull/72323. Depends on merging order (will need bt_hci entry in dts file).

Reply:
I can add a 
+zephyr,bt-hci = &esp32_bt_hci;
and
+&esp32_bt_hci{
+       status = "okay";
+};
(and update the defconfig) once  #72323 was committed. Adding it now will cause a device tree error due to an undefined label.

Reply:
Sounds good. Freeze window is June, 14th. Do you need this to be merged in v3.7.0?

Reply:
Hi Sylvio,
yes, please try to merge it before the v.3.7.0 merge window closes.